slackbot/slackclient.py

```diff
@@ -130,6 +130,12 @@ def send_message(self, channel, message, attachments=None, as_user=True):
     def get_channel(self, channel_id):
         return Channel(self, self.channels[channel_id])
 
+    def get_user(self, userid):
+        user = self.webapi.users.info(userid)
+        if not user.successful:
+            raise RuntimeError(user.error)
+        return user.body["user"]
```

Comment:
@jtatum has written the detailed steps to run the tests in a dedicated slack team, you can read it here: https://github.com/lins05/slackbot/blob/develop/CONTRIBUTING.md#configure-tests